### ISSUE TITLE
Multiline block quote loses linefeed in PDF

### ISSUE THREAD

**Issue by dselman** [accordproject/markdown-transform#299]
When a contract contains multiline block quotes the line feeds between lines are lost in the generated PDF.

**To Reproduce**
1. Convert the attached markdown file to a PDF:

[contract-5f4513931f0dca001b5aff0c-2020-08-25T13_37_44.813Z.md.zip](https://github.com/accordproject/markdown-transform/files/5124043/contract-5f4513931f0dca001b5aff0c-2020-08-25T13_37_44.813Z.md.zip)

![Screenshot 2020-08-25 at 14 36 10](https://user-images.githubusercontent.com/623108/91181294-a98eeb00-e6e0-11ea-9fc3-3e61e03863e8.png)

The PDF loses the linefeeds:

![Screenshot 2020-08-25 at 14 36 03](https://user-images.githubusercontent.com/623108/91181328-b3b0e980-e6e0-11ea-9e54-8d3072ec64a6.png)

**Expected behavior**
Linefeeds in the PDF should match the line feeds in the editor.

**Comment by jeromesimeon** [comment]
Not fully substantiated claim: this is an issue with the editor not with the pdf rendering.

<img width="1245" alt="Screenshot 2020-08-25 at 10 14 35 AM" src="https://user-images.githubusercontent.com/670099/91185485-e399c600-e6bb-11ea-8a88-9b213561a577.png">

**Comment by jeromesimeon** [comment]
With hardbreaks:

<img width="1249" alt="Screenshot 2020-08-25 at 10 16 21 AM" src="https://user-images.githubusercontent.com/670099/91185605-0d52ed00-e6bc-11ea-9367-3f3ecde73b67.png">

```
bash-3.2$ more foo.md 
This is a quote

> foo\
> foo\
> foo

more text
bash-3.2$ markus transform --from markdown --to pdf --input foo.md --output foo.pdf 
markus transform --from markdown --to pdf --input foo.md --output foo.pdf 
10:19:08 AM - info: Creating file: foo.pdf
10:19:08 AM - info: 
<binary pdf data>
```

[foo.pdf](https://github.com/accordproject/markdown-transform/files/5124240/foo.pdf)

**Comment by jeromesimeon** [comment]
See also: https://github.com/accordproject/web-components/issues/153

**Comment by jeromesimeon** [comment]
I should have looked inside the markdown file. This definitely should be rendered in pdf as two paragraphs:
<img width="1255" alt="Screenshot 2020-08-25 at 10 38 15 AM" src="https://user-images.githubusercontent.com/670099/91188272-1c876a00-e6bf-11ea-8e2c-a1e7e7ef32e7.png">

**Comment by jeromesimeon** [comment]
Fixed in #307
